### DISCUSSION TITLE
How do I make it so I can add my 8 team members as reviewers for my Pull Request easily?  Can this be done automatically?

### DISCUSSION THREAD

**Discussion by nicholaspshaw** [community/community#57241] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello all,

I am seeking guidance on achieving something I thought would be quite simple.  In short, I belong to a team of say 8 members, each time I create a pull request, I want my team members to be added as reviewers to avoid the need to search for each one in the reviewers panel.

I should stress, Codeowners file is not a solution here, as I do not want it to be specific to files, I want it to relate to any/all files.

So I have created a team, with just two members at the moment to test this out and it doesn't appear to be working.  When I launch a PR, within the reviewers section, when i type organisation/team-name, nothing appears for selection?  I do see other team names so this seems the right approach, does it just take time for the teams to appear there?

When typing a comment, I can do @organisation/team-name and it works.

Auto-assigning

I have attempted to set this up, again it doesn't look to be working.  I've included an image of what I put in the code review settings for the team, perhaps I've got it wrong?  My expectation was that any time I launch a PR (because I'm in the team) it would automatically add the other team members as reviewers on the PR.  Is that expectation incorrect?

<img width="1087" alt="Screenshot 2023-06-05 at 16 05 43" src="https://github.com/community/community/assets/56936829/79b011be-8cdd-459c-bff0-0b61122bef7b">

**Comment by nicholaspshaw** [comment, marked as answer]
Could it be that I'm not an admin of the repository so this is why?  I'm obviously within the organization, including on github so didn't think this would be a problem?

This is indeed the answer, those teams are required to have write access to the relevant repositories.

**Reply by baxelrod-bdai** [reply]
Same issue here: https://github.com/orgs/community/discussions/57575
